src/java/org/apache/cassandra/service/accord/async/AsyncOperation.java

```diff
@@ -311,12 +315,28 @@ public void run()
         }
     }
 
+    private boolean preRun()
+    {
+        commandStore.checkInStoreThread();
+        try
+        {
+            return runInternal(true);
+        }
+        catch (Throwable t)
+        {
+            logger.error(String.format("Operation %s failed", this), t);
```

Comment:
```suggestion
            logger.error("Operation {} failed", this, t);
```

Seen the pattern of using `String.format` w/e a throwable is done by many people, but that isn't required in any of the loggers...

Reply:
yep, I don't think this was something I introduced (the + signs are deceptive, just moved around)